spec/features/indexing_tags_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Integration tests for expected behaviors of our Solr indexing choices, through
+#   our whole stack: tests create cocina objects with factories, write them
+#   to dor-services-app, index the new objects via dor-indexing-app and then use
+#   the Argo UI to test Solr behavior such as search results and facet values.
+#
+# tag tests need javascript for facet testing because they are so slow to load in production
+#   javascript possibly also needed for the edit tags modal
+#
+# rubocop:disable Capybara/ClickLinkOrButtonStyle
+RSpec.describe 'Indexing and search results for tags', :js do
+  let(:item) { FactoryBot.create_for_repository(:persisted_item) }
+  let(:solr_id) { item.externalIdentifier }
+  let(:project_tag) { 'Project : ARS 78s : broken' }
+  let(:non_project_tag) { 'willet : murder of crows : curlew' }
+  let(:project_tag2) { 'Project : jira-517' }
+  let(:blacklight_config) { CatalogController.blacklight_config }
+  let(:solr_conn) { blacklight_config.repository_class.new(blacklight_config).connection }
+
+  # I tried to do this as a before(:context) but after much teeth gnashing, I gave up
```

Comment:
Leaving behind a working test of some utility seems better than spending more hours debugging some rspec weirdness interacting with javascript or scoping or whatever the problem is.

This way, we have one costly setup and a whole bunch of functionality of our tag fields indexing to UI checked.

Reply:
before(:context) is like before(:all), while a plain before is a before(:each)